salt/modules/shadow.py

```diff
@@ -96,15 +96,19 @@ def set_inactdays(name, inactdays):
     pre_info = info(name)
     if inactdays == pre_info['inact']:
         return True
-    cmd = 'chage -I {0} {1}'.format(inactdays, name)
+    cmd = ['chage']
+    if root is not None and __grains__['kernel'] != 'AIX':
+        cmd.extend(('-R', root))
```

Comment:
I cannot find documentation for `-R` when used with `chage`. Could you please point me to some?

Reply:
Sure!

I can see it in my system (openSUSE) and in the source code:

https://github.com/shadow-maint/shadow/blob/master/src/chage.c#L159

According to blame was added 7 years ago:

https://github.com/shadow-maint/shadow/commit/0857837e64a5a45ef686806aa3f3694ff733d5b2

Reply:
Hrm. My worry here is that I'm seeing this in certain docs:

https://linux.die.net/man/1/chage

That said, perhaps we could just improve error checking down in L104 and we should be fine. Thoughts?

Reply:
I do not know what to say, looks like that was documented 7 years ago too, according to the previous commit.

I can see the same option in CentOS 7, that provides shadow-utils-4.1.5.1-24.

I do not know from where https://linux.die.net gets the information, or when was the last update, but I cannot find the same option in useradd, another tool form shadow-utils:

https://linux.die.net/man/8/useradd

Also note that there is a missing CONFIGURATION section at the end of the man page, that was added 9 years ago:

https://github.com/shadow-maint/shadow/commit/5d6c3143047c30c2c402405a98806bdd664b6bee

But more important, the same -R parameter is already in the salt code for useradd (that is also part of the shadow-utils package), added three years ago:

https://github.com/saltstack/salt/commit/1eb3b9c5e4ae3952c8198692e994997207733406

IMHO linux.dice.net is not a reliable source of information.

P.S according to https://www.die.net/ the man pages are from 2002

Reply:
> That said, perhaps we could just improve error checking down in L104 and we should be fine. Thoughts?

This makes me wonder what to do with the already -R in useradd, that was added the same date in a different commit:

https://github.com/shadow-maint/shadow/commit/cecae46ccfb85c2c4bf6e0fe0419d740ba1f2ecc